accel-pppd/radius/acct.c

```diff
@@ -313,6 +314,14 @@ int rad_acct_start(struct radius_pd_t *rpd)
 		goto out_err;
 	}
 
+	if (ses->ipv6_dp) {
```

Comment:
Shouldn't it go to https://github.com/accel-ppp/accel-ppp/blob/master/accel-pppd/radius/req.c#L187 ?